TicTacToeApp/Game.swift

```diff
@@ -0,0 +1,36 @@
+
+public class Game {
+
+  private var board: Board
+  private var currentPlayer: PlayerMark
+
+  public init() {
+    board = Board()
+    currentPlayer = PlayerMark.X
+  }
+
+  public func makeMove(boardSpace: Int) {
+    board = board.markSpace(boardSpace, mark: currentPlayer)
+    currentPlayer = otherPlayer()
+  }
+
+  public func getCurrentPlayer() -> PlayerMark {
```

Comment:
There is. I'll use that instead, barring it complicating the design somehow, in which case I'll bring it up in person.